app/core/log.py

```diff
@@ -1,13 +1,45 @@
 import logging
 import logging.config
 import queue
+from enum import Enum
 from logging.handlers import QueueHandler, QueueListener
 from pathlib import Path
 from typing import Any
 
+import uvicorn
+
 from app.core.config import Settings
 
 
+class ModifiedFormatter(uvicorn.logging.DefaultFormatter):
+    class console_color(Enum):
+        """Colors can be found here: https://talyian.github.io/ansicolors/"""
+
+        DEBUG = "\033[38;5;12m"
+        INFO = "\033[38;5;10m"
+        WARNING = "\033[38;5;11m"
+        CRITICAL = "\033[38;5;9m"
+        BOLD = "\033[1m"
+        END = "\033[0m"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(datefmt="%d-%b-%y %H:%M:%S")
+
+    def format(self, record: logging.LogRecord) -> str:
+        fmt = (
+            "%(asctime)s - %(name)s - "
+            + self.console_color.BOLD.value
+            + "%(levelname)s"
+            + self.console_color.END.value
+            + " - "
+            + self.console_color[record.levelname].value
+            + "%(message)s"
+            + self.console_color.END.value
+        )
+        formatter = logging.Formatter(fmt, self.datefmt)
```

Comment:
I'm not sure if this is really pertinent, but could we instantiate `logging.Formatter` instead of creating a new one for each `format()` call?

Reply:
The original solution was calling the super().format() method but even when setting self._fmt it wasn't able to use the modified format. That's the solution I found on [Stack Overflow](https://stackoverflow.com/questions/384076/how-can-i-color-python-logging-output) that allow to color only a certain part of the text